modules/swagger-codegen/src/main/resources/typescript-angular/apiInterface.mustache

```diff
@@ -14,7 +24,7 @@ import { Configuration }                                     from '../configurat
     */
 {{/description}}
 export interface {{classname}}Interface {
-    defaultHeaders: Headers;
+    defaultHeaders: {{#useHttpClient}}Http{{/useHttpClient}}Headers;
     configuration: Configuration;
     [others: string]: any;
```

Comment:
could you please also include the following change:
https://github.com/swagger-api/swagger-codegen/pull/7732/files#diff-0a6663407a0b68718b67567a38ca1b20R28

Reply:
I don't understand what you want me to include here... sorry.

Reply:
@feikede I suggest to change the line
`[others: string]: any;` to `{{^useHttpClient}}[others: string]: any;{{/useHttpClient}}`